### ISSUE TITLE
Implement dynamic rendering of an thread

### ISSUE THREAD

**Issue by titatititatita** [titatititatita/forum#5]
Basically, thread is just a post that may contain replies to it grouped on a single page. Each thread as well as its replies can be referenced by an identifier (can be just a numeric value). The thread opens on a separate page isolated from other threads. After #4 is closed, there is a need to clarify database representation of a thread.

Definition of done:

- Database model of a thread is clarified
- HTML template from #4 is rendered dynamically using data from DB

Out of scope:

- Creation of new threads
- Replying to threads
- Dynamic rendering of index page
